spec/apis/event-listeners.spec.js

```diff
@@ -0,0 +1,69 @@
+export function notStartedEventListeners() {
+	describe(`event listeners before single-spa is started :`, () => {
```

Comment:
This test is actually just testing a different case than the bug we were seeing, but one where the same problem would potentially have surfaced. It's when singleSpa is not even started so it's not trying unmount/mount any apps or anything.